pkg/api/api.go

```diff
@@ -179,7 +180,10 @@ func ConfigureAPI(treeID uint) {
 			Addr:     fmt.Sprintf("%v:%v", viper.GetString("redis_server.address"), viper.GetUint64("redis_server.port")),
 			Password: viper.GetString("redis_server.password"),
 			Network:  "tcp",
-			DB:       0, // default DB
+			TLSConfig: &tls.Config{
+				InsecureSkipVerify: viper.GetBool("redis_server.insecure-skip-verify"),
```

Comment:
I believe you'll need to define this flag in `cmd/rekor-server/app/root.go` in order for this to work

Reply:
Like such - https://github.com/sigstore/rekor/pull/1998/commits/fe9b0a507a2ef61451d4b2b0f23c61dc509fab20 ?